README.md

```diff
@@ -48,13 +48,13 @@ Clone the repository
 
 - To set up a new project from the template, run the command:
 
-  - `$ make run PACKAGE_NAME={com.your.package} PROJECT_NAME={your_project_name} APP_NAME="{Your App Name}"`
+  - `$ make run PACKAGE_NAME={com.your.package} PROJECT_NAME={your_project_name} APP_NAME="{Your App Name}" APP_VERSION={your_app_version} BUILD_NUMBER={your_build_number}`
 
   - Then clean the project: `$ fvm flutter clean`
 
   - Re-fetch the project: `$ fvm flutter pub get`
 
-- The project uses the package name, the app name and the project name of the template if `PACKAGE_NAME`, `APP_NAME` and `PROJECT_NAME` aren't specified.
+- The project uses the package name, the app name, the project name and the project version of the template if `PACKAGE_NAME`, `APP_NAME`, `PROJECT_NAME`, `APP_VERSION` and `BUILD_NUMBER` aren't specified.
```

Comment:
use a Table:

| name | is mandatory | rationale|
-------|--------------|----------
| PACKAGE_NAME | Yes | The application package name |